jscomp/syntax/ast_external_attributes.ml

```diff
@@ -707,19 +707,19 @@ let handle_attributes
           | [], `Nm_na,  _ -> Js_module_as_var external_module_name
           | _, `Nm_na, _ -> Js_module_as_fn {splice; external_module_name }
           | _, #bundle_source, #bundle_source ->
-            Bs_syntaxerr.err loc Conflict_ffi_attribute
+            Bs_syntaxerr.err loc (Conflict_ffi_attribute "You used two [@@bs __] FFI attributes that cannot be used together.")
```

Comment:
Is there a more specific error we could use here? I couldn't figure out source code that would hit this code path—what is an example of code that would produce the error?